docs/data-sources/schedule.md

```diff
@@ -0,0 +1,28 @@
+---
+page_title: "FireHydrant Data Source: firehydrant_schedule"
+---
+
+# firehydrant_schedule Data Source
+
+Use this data source to get information on schedules.
+
+## Example Usage
+
+Basic usage:
+```hcl
+data "firehydrant_schedule" "my-oncall-schedule" {
+  name = "Main Oncall Schedule"
```

Comment:
If the name changes in whatever integration is used, will this then fail to find that record?  Can/should we do anything to prevent this or make people aware?